pkg/client/gen.go

```diff
@@ -81,43 +74,32 @@ func (*SonobuoyClient) GenerateManifest(cfg *GenConfig) ([]byte, error) {
 	}
 
 	plugins := []*manifest.Manifest{}
-	if includes(cfg.Config.PluginSelections, e2ePluginName) {
-		plugins = append(plugins, e2eManifest(cfg))
-	}
-	if includes(cfg.Config.PluginSelections, systemdLogsName) {
-		plugins = append(plugins, systemdLogsManifest(cfg))
+	for _, v := range cfg.DynamicPlugins {
+		switch v {
+		case e2ePluginName:
+			plugins = append(plugins, e2eManifest(cfg))
+		case systemdLogsName:
+			plugins = append(plugins, systemdLogsManifest(cfg))
+		}
```

Comment:
This iterates the loop once but the values could end up in either order based on how they were in the initial slice.

Good that it is not preferential there, but maybe bad that we still just put them in listed order and list the dynamic ones first.

I think this needs a sort by name at the end. It would be well defined and also independent of where the plugin came from.